src/views/pages/Swap/Swap.js

```diff
@@ -313,8 +310,6 @@ const Swap = () => {
           assetSwap2.baseAmount,
         ),
       )
-    } else {
-      clearInputs()
```

Comment:
Removed the 3 above clearInputs (on no value) as this is handled via '0' being handed on all calc functions instead (this is better for UX especially when the user tries to get rid of the 0 value to start with another number or no number (less than 1 unit starting with a '.')